cfpb/cfgov-refresh
# New EnforcementActionsFilterPage allows sorting and filtering on date_filed

Currently, the Enforcement Actions page uses the `BrowseFilterablePage` type, which is hard-coded to use `published_date` as the value to sort and filter by. This commit creates a new `EnforcementActionsFilterPage` so that we can use `date_filed` as the date for sorting and filtering Enforcement Action Pages.

## Testing

1. Run this branch with a recent production database dump. Make sure to run the migrations.
2. In the Wagtail admin, navigate to CFGov > Policy & Compliance > Enforcement. 
3. Create a new page with the Enforcement Actions Filter Page type. It should be a sibling page of the current Enforcement Actions `BrowseFilterablePage`.
    1. In the "content" section, include the "Filterable List" block. For best results, configure it like the current Enforcement Actions page is configured:
<img width="552" alt="Screen Shot 2020-03-24 at 11 58 23 AM" src="https://user-images.githubusercontent.com/4295388/77448197-dc962100-6dc6-11ea-9a5a-6695ac939f57.png">

4. Take note of the page id of the new page (you can see it in the url of the edit page)
5. From the command line, run this command: `cfgov/manage.py move_pages 1327 [new_page_id]`
6. Visit your new page.
    - See that the Enforcement Actions are sorted in reverse chronological order by `date_filed`.
    - Use the date filtering to confirm that it is filtering on the `date_filed` value of the posts.

## Screenshots
<img width="590" alt="Screen Shot 2020-03-24 at 12 06 17 PM" src="https://user-images.githubusercontent.com/4295388/77449079-ea987180-6dc7-11ea-90ee-eac84798b6c4.png">


## Todos

- After this is merged and deployed, we will need to take some manual steps to get the content onto the new page type to get the desired behavior.

## Checklist
